lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -181,6 +156,11 @@ class FirstMeaningfulPaint extends Audit {
           !!navigationStart && event.ts >= navigationStart.ts) {
         firstContentfulPaint = event;
       }
+      if (event.name === 'firstMeaningfulPaint' && event.args.frame === mainFrameID &&
+          !!navigationStart && event.ts >= navigationStart.ts) {
+        firstMeaningfulPaint = event;
+      }
+
       // COMPAT: frame property requires Chrome 52 (r390306)
       // https://codereview.chromium.org/1922823003
       if (event.name === 'FrameView::performLayout' &&
```

Comment:
you can remove this `'FrameView::performLayout'` and the `event.name === 'Paint'` blocks.